xwiki-rendering-syntaxes/xwiki-rendering-syntax-xhtml/src/main/java/org/xwiki/rendering/internal/parser/xhtml/wikimodel/XHTMLXWikiGeneratorListener.java

```diff
@@ -145,4 +165,70 @@ private ResourceReference computeResourceReference(String rawReference)
 
         return reference;
     }
+
+    private boolean isMetaDataElement(WikiParameters parameters)
+    {
+        if (parameters.getParameter(CLASS_ATTRIBUTE) != null) {
+            return METADATA_CONTAINER_CLASS.equals(parameters.getParameter(CLASS_ATTRIBUTE).getValue());
+        }
+
+        return false;
```

Comment:
This can be written in a single line:

    return parameters.getParameter(CLASS_ATTRIBUTE) != null &&
        METADATA_CONTAINER_CLASS.equals(parameters.getParameter(CLASS_ATTRIBUTE).getValue());

Reply:
Indeed, I'll simplify